src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -7,8 +7,11 @@
 /**
  * Class to create the modBlocks.
  * References to the blocks can be made here
+ *
+ * We disabled the following finals since we are neither able to mark the items as final, nor do we want to provide public accessors.
  */
-public class ModBlocks
+@SuppressWarnings({"squid:ClassVariableVisibilityCheck", "squid:S2444", "squid:S1444", "squid:S1820" , })
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:ClassVariableVisibilityCheck, squid:S2444, squid:S1444, squid:S1820' warnings is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)